### PR TITLE
feat: expand user options for views with schedule

### PR DESCRIPTION
**Pull request by cpvalente**
small PR to add two new options to views with the scheduler component (backstage and public)

1. option to hide events that are in the past
2. option to stop the auto cycling of the schedule

### CODE REVIEW

File: apps/client/src/common/components/view-params-editor/constants.ts [hunk 1]
```diff
@@ -194,6 +194,37 @@ export const LOWER_THIRDS_OPTIONS: ParamField[] = [
...
+  {
+    id: 'hidePast',
+    title: 'Hide past events',
+    description: 'Scheduler will only show upcoming events',
+    type: 'boolean',
+  },
+  {
+    id: 'stopCycle',
+    title: 'Stop cycling through event pages',
+    description: 'Schedule will auto not cycle through events',
```
Comment (alex-Arc) [inline apps/client/src/common/components/view-params-editor/constants.ts:208]: Schedule will not auto cycle through events

**Review by alex-Arc — CHANGES_REQUESTED** [review]
'hide events that are in the past' dose not work when the publicEventId is null but that seams to be a problem upstream

**Comment by cpvalente** [comment]
> 'hide events that are in the past' dose not work when the publicEventId is null but that seams to be a problem upstream

If `publicEventId` is null we do not have a running event. So there are no events in the past.
Is this not expected?

**Comment by alex-Arc** [comment]
here all events are public and event 3 is loaded and hide past is enabled in both the backstage and public view
image

backstage hides event 1 -> 3 but public shows them all

if i add a console log to ScheduleContext.tsx the public one cant find the evet
image

File: apps/client/src/common/components/view-params-editor/constants.ts (hunk 1 above)
Comment (cpvalente) [reply apps/client/src/common/components/view-params-editor/constants.ts:208]: good catch, thank you

**Comment by cpvalente** [comment]
good catch @alex-Arc, I hope I have managed to cover them all now

**Review by alex-Arc — APPROVED** [review]

**Pull request merged**